.gitignore

```diff
@@ -71,3 +71,6 @@ tags
 docs/
 index.md
 about_pkgdown_documentation.md
+
+# Files generated by testthat
+.rds
```

Comment:
It might be better to make this more specific. An `rds` file is a binary file containing a single R object, as created by `saveRDS`. It's possible that the package could contain legitimate `rds` files that should be tracked by `git`, such as in the `data` directory or elsewhere in the `tests` directory (representing saved output from a function).